packages/services/schema/src/lib/compose.ts

```diff
@@ -25,14 +25,13 @@ interface BrokerPayload {
   body: string;
 }
 
-const EXTERNAL_COMPOSITION_RESULT = z.union([
+const ExternalCompositionResultModel = z.union([
   z.object({
     type: z.literal('success'),
     result: z.object({
       supergraph: z.string(),
       sdl: z.string(),
     }),
-    includesNetworkError: z.boolean().default(false),
```

Comment:
We never want to receive this from the external composition service. It i state that only exists on our end.